packages/sdk/src/modules/JsonRpcDatasource.ts

```diff
@@ -21,28 +22,37 @@ export default class JsonRpcDatasource extends BaseDatasource {
       throw new Error("Invalid request")
     }
 
-    return rpc[this.network].call<number>("GetBalance", { address }, rpc.id)
+    return rpc[this.network].call<number>("Address.GetBalance", { address }, rpc.id)
   }
 
   async getInscription(id: string, decodeMetadata = false) {
     if (!id) {
       throw new Error("Invalid request")
     }
 
-    let inscription = await rpc[this.network].call<Inscription>("GetInscription", { id }, rpc.id)
+    let inscription = await rpc[this.network].call<Inscription>("Ordinals.GetInscription", { id }, rpc.id)
     if (decodeMetadata) {
       inscription = DatasourceUtility.transformInscriptions([inscription])[0]
     }
 
     return inscription
   }
 
-  async getInscriptions(outpoint: string, decodeMetadata = false) {
-    if (!outpoint) {
-      throw new Error("Invalid options provided.")
+  async getInscriptions(owner: string, decodeMetadata = false, next: string | null = null) {
+    if (!owner) {
+      throw new Error("Invalid request")
     }
 
-    const inscriptions = await rpc[this.network].call<Inscription[]>("GetInscriptions", { outpoint }, rpc.id)
+    let inscriptions: Inscription[] = []
+    do {
+      const { inscriptions: _inscriptions, pagination } = await rpc[this.network].call<{
+        inscriptions: Inscription[]
+        pagination: JsonRpcPagination
+      }>("Ordinals.GetInscriptions", { filter: { owner }, pagination: { limit: 25, next } }, rpc.id)
+      inscriptions = inscriptions.concat(_inscriptions)
+      next = pagination.next
+    } while (next !== null)
+
```

Comment:
does this mean we can't get inscriptions from the outpoint anymore? only from inscriptionId?

Reply:
No. This looks like a mistake. 

Outpoint will be restored and more filtering props will be added to this fn. Good catch.

Reply:
added more filters: https://github.com/sadoprotocol/ordit-sdk/pull/69/commits/0ce854e0715a3489518db072971b16f44ba1ebad